### DISCUSSION TITLE
Failed to parse SMTP response(?)

### DISCUSSION THREAD

**Discussion by SummonHIM** [authelia/authelia#8457] [Q&A] [answered]
```
time="2024-12-16T07:44:27Z" level=error msg="Error occurred creating user session elevation One-Time Code challenge for user 'summonhim': error occurred sending the user the notification" error="notifier: smtp: failed to send message: sending SMTP RESET command: failed to send RSET to SMTP client: short response: \x00\x00\x00\x18\x00\x00\x00\b\x00\x00\x00\x00\x00\x00\x00\x00250 OK, affected message ID: <xxx@qq.com>" method=POST path=/api/user/session/elevation remote_ip=127.0.0.1
```

My smtp provider return a hex(?) response 250 OK. But authelia considered it an error. how to fix it?

**Comment by SummonHIM** [comment]
Email sending is normal and I can receive emails

**Reply by Crowley723** [reply, marked as answer]
At a quick glance it seems like there is some difference in the format of the message that Authelia is getting.

I would suggest you try with a different SMTP provider (Google is pretty easy) just to rule out the SMTP provider being the issue.

**Reply by SummonHIM** [reply]
Thank you for your suggestion, but in my country, we are unable to use Gmail normally. If we use Outlook, the account may get banned due to triggering automatic sending limits.

**Reply by james-d-elliott** [reply]
The idea is correct. You need to set it up with another SMTP server to test. Be one of the mentioned ones or something like postfix or stalwart.

To me the only fix can be made on the mail server side as they're not respecting the mail [specifications](https://www.rfc-editor.org/rfc/rfc5321#section-4.1.1.5), the `250 OK` must be sent unconditionally on its own, not padded with many null bytes and an escape byte.
